docs/subvolume.md

```diff
@@ -0,0 +1,53 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete <subvolumes> <filesystem> <subvolumegroup>`:
+    [delete](#delete) stale subvolumes after user's confirmation.
+    It will list and delete only the stale subvolumes to prevent any loss of data.
+    * subvolumes: comma-separated list of subvolumes of same filesystem and subvolumegroup.
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# Listing subvolumes
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110004 csi
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110005 csi
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110006 csi
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110007 csi
+
+```
+
+```bash
+kubectl rook-ceph subvolume ls --stale
+
+# Listing stale subvolumes
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110004 csi
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110005 csi
+
+```
+
+## delete
+
+```bash
+kubectl rook-ceph subvolume delete csi-vol-427774b4-340b-11ed-8d66-0242ac110004 ocs-storagecluster csi
+
+# subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
+
+```
+
+```bash
+kubectl rook-ceph subvolume delete csi-vol-427774b4-340b-11ed-8d66-0242ac110004,csi-vol-427774b4-340b-11ed-8d66-0242ac110005 ocs-storagecluster csi
+
+# subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
```

Comment:
The CI output shows that all the output has the "Info" prefix. This sample output needs to match the actual output format.
```suggestion
# Info: subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
```